programs/bpf_loader/src/lib.rs

```diff
@@ -913,69 +989,20 @@ fn process_loader_upgradeable_instruction(
             ic_logger_msg!(log_collector, "Upgraded program {:?}", new_program_id);
         }
         UpgradeableLoaderInstruction::SetAuthority => {
-            instruction_context.check_number_of_instruction_accounts(2)?;
-            let mut account =
-                instruction_context.try_borrow_instruction_account(transaction_context, 0)?;
-            let present_authority_key = transaction_context.get_key_of_account_at_index(
-                instruction_context.get_index_of_instruction_account_in_transaction(1)?,
+            process_set_authority(
+                instruction_context,
+                transaction_context,
+                &log_collector,
+                false,
+            )?;
+        }
+        UpgradeableLoaderInstruction::SetAuthorityChecked => {
```

Comment:
This new instruction needs to be behind a feature gate. You can search for how `StakeInstruction::InitializeChecked` does this for an example

Reply:
looking at the instructions in [feature_set.rs](https://github.com/solana-labs/solana/blob/master/sdk/src/feature_set.rs#L11), it says a core contributor should hold the keypair needed for the feature set pubkey. do you mind generating one for me?